openedx/stanford/djangoapps/register_cme/admin.py

```diff
@@ -5,4 +5,35 @@
 
 from .models import ExtraInfo
 
-admin.site.register(ExtraInfo)
+
+class ExtraInfoAdmin(admin.ModelAdmin):
+    """
+    Admin interface for ExtraInfo model.
+    """
+
+    readonly_fields = (
```

Comment:
Let's alphabetize these fields (move this between `list_display` and `search_fields`).